tornjak-backend/cmd/agent/agent.go

```diff
@@ -205,3 +208,34 @@ func getSocketPath(config *run.Config) string {
 
 	return "unix://" + socketPath
 }
+
+// below copied from spire/cmd/spire-server/cli/run/run.go, but with TornjakConfig
+func parseTornjakConfig(path string) (*agentapi.TornjakConfig, error) {
+	c := &agentapi.TornjakConfig{}
+
+	if path == "" {
+		return nil, errors.New("Bad Tornjak config file: path given is empty")
+	}
+
+	// friendly error if file is missing
+	byteData, err := os.ReadFile(path)
+	if os.IsNotExist(err) {
```

Comment:
usually the pattern for this is to check if the err is not nil before using it.. can check on the contract to make sure it holds up for nil